components/console-backend-service/OWNERS

```diff
@@ -0,0 +1,11 @@
+reviewers:
+  - console-approvers
+  - michal-hudy 
```

Comment:
As I rarely contribute to this project, please remove me from reviewers.

Reply:
done